<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermianlOption</class>
 <widget class="QWidget" name="TermianlOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Family:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="familyComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sizeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Zoom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fontZoomSpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="antialiasCheckBox">
        <property name="text">
         <string>Antialias</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
